sunpy/data/test/adapt.header

```diff
@@ -0,0 +1,75 @@
+SIMPLE  =                    T / file does conform to FITS standard
+BITPIX  =                  -32 / number of bits per data pixel
+NAXIS   =                    3 / number of data axes
+NAXIS1  =                  360 / length of data axis 1
+NAXIS2  =                  180 / length of data axis 2
+NAXIS3  =                   12 / length of data axis 3
```

Comment:
So in our test data, this will result in a 2D map. In the actual ADAPT files, is there a 3rd dimension? If so, I think the Map will complain about this when it drops the third dimension. There is also the question of how we would support taking out dimensions that are not just the 0th dimension.

Reply:
This is kind of odd because this is a dimension in the WCS that has no meaning when it comes to the WCS, i.e. you cannot define a `CTYPE` or a `CDELT` for the "ensemble axis"

Reply:
Yes I agree, I need to get a real file and see what it looks like.

But it lacks the complete set of FITS keywords for the third axis.

Reply:
Yeah it's really bizarre...